### ISSUE TITLE
update HBA bridge mappings to nerves to point to nerve roots

### ISSUE THREAD

**Issue by dosumis** [obophenotype/uberon#3264]
CC @gouttegd  - Can you confirm how we should update Allen Brain Atlas bridge files now.  (I should probably know this!)

- We may also need to check whether all ABA terms are being classified as brain region.

**Comment by gouttegd** [comment]
All bridge files (including the custom ones like the MBA and DMBA bridges) are updated with a single command:

```sh
sh run.sh make refresh-bridges
```

This is normally done by @anitacaron (as part of the refresh of all external resources) ahead of a release, but can be done anytime by anyone if there’s a need.

**Comment by gouttegd** [comment]
Wait, do you mean updating the _contents_ of the bridges?

It depends on the bridge. For the MBA and DMBA bridges, they are entirely managed outside of Uberon, so they should be updated in their respective repositories (https://github.com/brain-bican/mouse_brain_atlas_ontology and https://github.com/brain-bican/developing_mouse_brain_atlas_ontology).

For the HBA, DHBA, and PBA bridges, the source of truth is still the Uberon `-edit` file, more precisely the cross-references carried by Uberon term.

**Comment by gouttegd** [comment]
So for example, Uberon’s `hypoglossal nerve` (UBERON:0001650) is currently mapped to HBA:9343. To remap the HBA term to Uberon’s `hypoglossal nerve root` (UBERON:0004675), you need to remove the `HBA:9343` cross-reference on UBERON:0001650 and put it on UBERON:0004675 instead.

**Comment by dosumis** [comment]
@AvolaAmg  - can you do these edits to Uberon?  Would be good to do soon if poss and work with @aleixpuigb to make an Uberon release as this affects our ability to do PCL updates.
@hkir-dev  - can you provide a list of nerve terms in HBA?

**Comment by hkir-dev** [comment]
|Nerves|
|----|
https://purl.brain-bican.org/ontology/hbao/HBA_9337
https://purl.brain-bican.org/ontology/hbao/HBA_9328
https://purl.brain-bican.org/ontology/hbao/HBA_9319
https://purl.brain-bican.org/ontology/hbao/HBA_9325
https://purl.brain-bican.org/ontology/hbao/HBA_9300
https://purl.brain-bican.org/ontology/hbao/HBA_9322
https://purl.brain-bican.org/ontology/hbao/HBA_9343
https://purl.brain-bican.org/ontology/hbao/HBA_9334
https://purl.brain-bican.org/ontology/hbao/HBA_9316
https://purl.brain-bican.org/ontology/hbao/HBA_9331
https://purl.brain-bican.org/ontology/hbao/HBA_9340
https://purl.brain-bican.org/ontology/hbao/HBA_9299

Explanation of unsats:  [unsats.md](https://github.com/obophenotype/uberon/files/15143511/unsats.md)

@AvolaAmg you can run `make` command in this folder ([hba_test.zip](https://github.com/obophenotype/uberon/files/15143512/hba_test.zip)) to reproduce the issue. Then you can locally edit the bridge file and see if unsats are fixed. Please slack me for any issue.

**Comment by cmungall** [comment]
Can you summarize some commonalities between these? I expect there is a
common cause. (I have found simply feeding unsats.md into long-context
model AIs like claude to be a fast way of getting at underlying causes)

**Comment by dosumis** [comment]
I think you have to read these from the perspective of how the annotated domains relate to each other on a 3D brain.  I'll try to pull out an image to post.

**Comment by aleixpuigb** [comment]
> @AvolaAmg - can you make these edits to Uberon? Would be good to do soon if poss and work with @aleixpuigb to make an Uberon release as this affects our ability to do PCL updates.

@anitacaron is planning to do an Uberon release soon (probably this week). @AvolaAmg, do you need help with this ticket to make sure it gets in this release?

**Comment by AvolaAmg** [comment]
Hi @aleixpuigb thanks for your offer, I will have a look at this ticket tomorrow first thing and let you know if I need any help.
